### ISSUE TITLE
[BUG] - `Button` and `Link` components in variant `ghost` are missing an outline in high-contrast-mode

### ISSUE THREAD

**Issue by o-hook** [porsche-design-system/porsche-design-system#3581]
### Description

The `Button`and `Link`components in variant `ghost` are missing an visual outline when the document is set in high-contrast-mode (HCM). This makes it hard to distinguish those components as clickable UI elements.

<img width="632" alt="Bildschirmfoto 2024-10-30 um 08 41 47" src="https://github.com/user-attachments/assets/45867322-65fe-46a2-b7e3-f1a4736e65c0">

### JS Framework

all

### Porsche Design System version

3.20.0

### On which device(s) you can see the bug?

all

### In which browsers you can see the bug?

all

### Browser version

all

### Reproduction Example

Storefront

### Steps to reproduce the behavior

1. Go to https://designsystem.porsche.com/v3/components/button/examples or https://designsystem.porsche.com/v3/components/link/examples
2. Activate emulation of forced-colors in Chrome
3. See error: `Button` and `Link` component in variant `ghost` have no visual outline

### Expected behavior

Though the `ghost`variant of the `Button`and `Link` components have in general no outline, they need one in HCM to be easily distinguished as UI elements.
